### ISSUE TITLE
Sonoff devices unavailable

### ISSUE THREAD

**Issue by basakmurat2005** [AlexxIT/SonoffLAN#1402]
Hi, 
After updating the latesr version, all devicesi are gone offline.
reloading integration but no change
!--
- Write issue ONLY in English
- Search if similar issue already exist, also check closed issues, DO NOT CREATE DUPLICATES
- Check Integration errors in Hass logs (Configuration > Logs), maybe answer there
- Check Integration debug (readme) for something useful
- Read the readme carefully, maybe the answer is there
-->

**Comment by mahdi-m1** [comment]
Same

**Comment by clintferns2911** [comment]
Yes unfortunately having the same issue. If i redownload the integration, it'll work for sometime and then go offline again

**Comment by gosoftch** [comment]
Same ...

**Comment by GabrielTan7771** [comment]
I have been facing similar issues more than 20hrs ago. Likewise, ssl certificate error.  I have been searching all over to resolve but all came up empty.

**Comment by AlexxIT** [comment]
No one other than @froias16 has provided anything useful for diagnosis.
https://github.com/AlexxIT/SonoffLAN?tab=readme-ov-file#issues

@froias16 I can confirm that the Asian server forgot to update the certificate. The other servers haven't forgotten. You can report to ewelink support. It's their mistake.

**Comment by froias16** [comment]
> Logger: custom_components.sonoff.core.ewelink.cloud Source: custom_components/sonoff/core/ewelink/cloud.py:588 integration: Sonoff (documentation, issues) First occurred: 2:20:14 PM (5 occurrences) Last logged: 2:24:01 PM
> 
> Cloud WS Connection error: Cannot connect to host as-dispa.coolkit.cc:443 ssl:True [SSLCertVerificationError: (1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:1000)')]

Have sent an email to ewelink support. Hope it gets picked up and resolved the soonest.

Thanks @AlexxIT for the advise.

**Comment by chayan1983** [comment]
I'm reporting from Turkey. There is access via the Ewelink app. Only Zigbee devices do not work on Home Assistant.

**Comment by safvan89** [comment]
is there any option available to change server from asia to somewhere else?

**Comment by Ngagu** [comment]
Same, all my sonoff devices appear Unavailable but still can access via eweLink app.

**Comment by froias16** [comment]
<img width="746" alt="ewelink support" src="https://github.com/AlexxIT/SonoffLAN/assets/85502093/a6b41096-33b8-4122-8b2c-45821d5a43d1">

Issue seems resolved after the last response I got from Ewelink Support

**Comment by Suleiman700** [comment]
I had this issue since this morning (around 10am)
It seems the issue has fixed

**Comment by gosoftch** [comment]
I'm not at home and really not nearby...
For me my Sonoff is still "unavailable". Did you have to do a reset or something like that or did everything come back automatically?

**Comment by clintferns2911** [comment]
Fixed for me as well. Needed to restart HA
Thanks to all.

**Comment by basakmurat2005** [comment]
Mine is ok right now. Thanx for the quick response and your cooperation. @AlexxIT
